app/MMB/src/scenes/list/cityImage/StatusBadge.js

```diff
@@ -17,11 +17,19 @@ const StatusBadge = (props: Props) => {
   const isPastBooking = idx(props.data, _ => _.isPastBooking);
 
   switch (status) {
-    case 'REFUNDED':
+    case 'CANCELLED':
```

Comment:
Overall I think this file can be much, much smaller. What do you think of an object like:

```js
const statuses = {
   // Map object
   CANCELLED: {
     translation: 'your-value-here'
     style: 'your-value-here'
   }
  // Other status
  ...
}
```

`textStyle` is the same for all. Then you can do.

```js
switch(status) {
 return (
    <AdaptableBadge
        translation={<Translation id={statuses[STATUS].translation} />}
        style={statuses[STATUS].style}
        textStyle={styles.badgeText}
    />
 )
}
```

So you only write the `AdaptableBadge` component once.